webxdc.js

```diff
@@ -51,6 +51,9 @@ window.webxdc = (() => {
             console.log('[Webxdc] description="' + description + '", ' + JSON.stringify(_update));
             updateListener(_update);
         },
+        sendToChat: (content) => {
```

Comment:
you can copy what I did in the simulator for the calendar app https://github.com/webxdc/calendar/pull/11/files#diff-4dc52a56e760eb8eebd14247c25417f6a9a8a0d7cd905787a088b169d6ef197f

Reply:
I think the right place to update is in the [hello](https://github.com/webxdc/hello) repo, that is the template where [webxdc.js](https://github.com/webxdc/hello/blob/master/webxdc.js) mini-emulator and [webxdc.d.ts](https://github.com/webxdc/hello/blob/master/webxdc.d.ts) are maintained, then just copy+paste the files in the other places that need it

Reply:
created an issue: https://github.com/webxdc/hello/issues/45

probably the other advanced emulator also needs updating, not sure if it is a matter to also copying the webxdc.js file from the hello project